### PR TITLE
Add 'lager' package

### PR DESCRIPTION
**Pull request by Bjoe**
* I've followed [this guide](https://cpp-pm-hunter.readthedocs.io/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/Bjoe/hunter-t1t06/builds/29874306
  * https://travis-ci.org/Bjoe/hunter/builds/632247904

* I have submitted CI configs to https://github.com/cpp-pm/hunter-testing targeting `pkg.template` branch,
  see this merged pull request https://github.com/cpp-pm/hunter-testing/pull/29

### CODE REVIEW

File: cmake/projects/Lager/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    Lager
+    VERSION
+    0.0.1-dbc1fde
+    URL
+    "https://github.com/Bjoe/lager/archive/68e69c30667c12e01ef98a65e3ecef12c461f2e1.zip"
```
Comment (Bjoe) [inline cmake/projects/Lager/hunter.cmake]: It need to have a repo in cpp-pm

Comment (rbsheth) [reply cmake/projects/Lager/hunter.cmake]: Done https://github.com/cpp-pm/lager

Comment (Bjoe) [reply cmake/projects/Lager/hunter.cmake]: Unfortunately I pushed my `hunter` branch to  https://github.com/cpp-pm/lager repo and then I saw your comment on `cppast` PR and rename this branch. But I cannot delete the `hunter` branch in https://github.com/cpp-pm/lager repo. Can you please delete this branch?

**Comment by Bjoe** [comment]
With the new version number:

I've tested this package remotely and have excluded all broken builds.
Here is the links to the Travis/AppVeyor with status "All passed":

* https://travis-ci.org/Bjoe/hunter/builds/633612486
* https://ci.appveyor.com/project/Bjoe/hunter-t1t06/builds/29938255

File: cmake/projects/Lager/hunter.cmake (hunk 1 above)
Comment (rbsheth) [reply cmake/projects/Lager/hunter.cmake]: Done

**Comment by rbsheth** [comment]
Released: https://github.com/cpp-pm/hunter/releases/tag/v0.23.241

**Pull request merged**